Airports/Other/EGFE.txt

```diff
@@ -1,3 +1,3 @@
 Haverfordwest
 N051.49.59.000 W004.57.40.000
-122.200
\ No newline at end of file
+122.605
```

Comment:
```suggestion
122.600
```

Sorry, the issue could've been clearer that although the frequency is unavailable due 8.33, we should still put 122.600 in here :)